### PR TITLE
Improve appearance of scheduling page

### PR DESCRIPTION
**Pull request by sjhuang26**
* make schedule buttons consistent with rest of UI
* refactor schedule-menu to use flexbox
* if you hover over an overflowing schedule event, a scrollbar will appear
image

### CODE REVIEW

**Comment by Bad-Science** [comment]
Thanks for the PR @sjhuang26 !

I think these buttons are an improvement and I like using flexbox here.

I also like being able to handle the overflowing schedule events, but rather than having scrollbars appear, could we make it so they simply expand as necessary when you hover over them? I think that would be a little cleaner.

**Review by Bad-Science — CHANGES_REQUESTED** [review]
Thanks for the PR! See above comments for request changes

**Comment by sjhuang26** [comment]
Requested changes have been made.

File: src/app/schedule-view/schedule-event/component.ts [hunk 1]
```diff
@@ -10,8 +10,23 @@ import { ScheduleEvent } from '../../models/schedule-event.model';
...
\ No newline at end of file
+    @Input() normalHeight: number;
+
+    hover: boolean = false;
+
+    changeHover(newHover: boolean) {
+        this.hover = newHover;
+    }
+    getHeight() {
+        return this.hover ? 'auto' : this.normalHeight + 'px';
```
Comment (Bad-Science) [inline src/app/schedule-view/schedule-event/component.ts]: Great thanks! Looks good, can we just add (return) type annotations to these functions, and use two-space indentation @sjhuang26 ? Then I will merge this.

**Comment by sjhuang26** [comment]
More requested changes have been made.

File: src/app/schedule-view/schedule-event/component.ts [hunk 2]
```diff
@@ -10,8 +10,25 @@ import { ScheduleEvent } from '../../models/schedule-event.model';
...
\ No newline at end of file
+  @Input() normalHeight: number;
+
+  hover: boolean = false;
+
+  changeHover(newHover: boolean): void {
+    this.hover = newHover;
+  }
+
+  getHeight(): number {
```
Comment (Bad-Science) [inline src/app/schedule-view/schedule-event/component.ts]: this returns a string not a number, right?

**Review by Bad-Science — APPROVED** [review]

**Comment by Bad-Science** [comment]
LGTM, thanks @sjhuang26 !

**Pull request merged**